app/models/concerns/resettable.rb

```diff
@@ -0,0 +1,24 @@
+module Resettable
```

Comment:
Could you namespace this under concerns/vacancy/resettable? The module will be called Vacancy::Resettable. I ask because I'm creating the vacancy directory in my open PR.